vnext/local-cli/src/runWindows/runWindowsOptions.ts

```diff
@@ -146,5 +149,35 @@ export const runWindowsOptions: CommandOption[] = [
   {
     name: '--direct-debugging [number]',
     description: 'Enable direct debugging on specified port',
+    parse: parseDirectDebuggingPort,
   },
 ];
+
+function parseBuildArch(arg: string): BuildArch {
+  const supportedArches: BuildArch[] = ['x86', 'x64', 'ARM64', 'ARM'];
+  for (const supported of supportedArches) {
+    if (arg.toLowerCase() === supported.toLowerCase()) {
+      return supported;
+    }
+  }
+
+  errorOut(`Unrecognized --arch '${arg}'. Expected one of ${supportedArches}`);
+}
+
+function parseDirectDebuggingPort(arg: string): number {
+  const num = parseInt(arg, 10);
+
+  if (!Number.isInteger(num)) {
+    errorOut(`Expected argument '--direct-debugging' to be a number`);
+  }
+  if (num < 1024 || num >= 65535) {
```

Comment:
This is slightly different from the old check bc a quick googling says 1023 is reserved, but 1024 isn't.